class-tgm-plugin-activation.php

```diff
@@ -1128,6 +1108,75 @@ public function _get_plugin_data_from_name( $name, $data = 'slug' ) {
 		}
 
 		/**
+		 * Retrieve the download url for a package.
+		 *
+		 * @since 2.5.0
+		 *
+		 * @param string $slug
+		 *
+		 * @return string Plugin download URL or path to local file
+		 */
+		public function get_download_url( $slug ) {
+			$dl = '';
```

Comment:
Scrutinizer will likely pick up on this as being too short a variable name. `$download_url` or `$url` would be a little more descriptive.